contracts/strategies/idle/IdleFinanceStrategy.sol

```diff
@@ -230,13 +243,33 @@ contract IdleFinanceStrategy is IStrategy, RewardTokenProfitNotifier {
     return invested.add(IERC20(underlying).balanceOf(address(this)));
   }
 
-  function setLiquidation(bool _sellComp, bool _sellIdle, bool _claimAllowed) public onlyGovernance {
-    sellComp = _sellComp;
-    sellIdle = _sellIdle;
+  function setLiquidation(address _token, bool _sell) public onlyGovernance {
+     sell[_token] = _sell;
+  }
+
+  function setClaimAllowed(bool _claimAllowed) public onlyGovernance {
     claimAllowed = _claimAllowed;
   }
 
   function setProtected(bool _protected) public onlyGovernance {
     protected = _protected;
   }
+
+  function setMultiSig(address _address) public onlyGovernance {
+    multiSig = _address;
+  }
+
+  function setRewardClaimable(bool flag) public onlyGovernance {
```

Comment:
And for this as well (as well as liquidation paths). Basically, let's make sure that no additional calls would be necessary when deploying these strategies (otherwise we keep forgetting).

Reply:
Included in the `...Mainnet` strategies.